styleguide/_themes/derek/scss/patterns/announcements.scss

```diff
@@ -0,0 +1,88 @@
+.announcement-container {
+  background-color: $white;
+  border-bottom: 1px solid $light-gray;
+  border-right: 1px solid $light-gray;
+  border-top: 1px solid $light-gray;
+  margin-bottom: 10px;
+  overflow: hidden;
+  padding: 10px;
+  position: relative;
+  width: $full-width;
+}
+
+.announcement-icon {
+  @include make-md-column(2);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 60px 60px;
+  min-height: 100px;
+  opacity: .7;
+  padding: 40px;
+  text-align: center;
+}
```

Comment:
Let's talk about utilizing the flex implementation we use for features and additional resources instead of setting fixed heights and bg sizes ect.